src/java.desktop/macosx/native/libawt_lwawt/awt/CRobot.m

```diff
@@ -63,8 +63,13 @@
 static int gsEventNumber;
 static int* gsButtonEventNumber;
 static NSTimeInterval gNextKeyEventTime;
+static CGEventFlags initFlags;
+static int allModifiersMask = kCGEventFlagMaskShift | kCGEventFlagMaskControl
+                              | kCGEventFlagMaskAlternate | kCGEventFlagMaskCommand
+                              | kCGEventFlagMaskAlphaShift;
```

Comment:
You need to add kCGEventFlagMaskSecondaryFn here otherwise the "secondary fn fix" might catch regression.

Reply:
Updated.